core/pta/remote_attestation/sub.mk

```diff
@@ -0,0 +1,4 @@
+srcs-$(CFG_VERAISON_ATTESTATION_PTA) += remote_attestation.c
```

Comment:
This `sub.mk` file needs `cflags-y += -Wno-declaration-after-statement` and `cflags-y += -Wno-redundant-decls`.

Reply:
DONE